src/editor/panel.ts

```diff
@@ -1,19 +1,32 @@
-import { INotebookModel } from '@jupyterlab/notebook';
+import {
+  INotebookModel,
+  NotebookPanel,
+  StaticNotebook
+} from '@jupyterlab/notebook';
 
-import { Context } from '@jupyterlab/docregistry';
+import {
+  CodeCell,
+  CodeCellModel,
+  MarkdownCell,
+  MarkdownCellModel
+} from '@jupyterlab/cells';
 
-import * as nbformat from '@jupyterlab/nbformat';
+import { IEditorMimeTypeService } from '@jupyterlab/codeeditor';
+
+import { IRenderMimeRegistry } from '@jupyterlab/rendermime';
+
+import { DocumentRegistry } from '@jupyterlab/docregistry';
 
 import { ReadonlyPartialJSONValue } from '@lumino/coreutils';
 
 import { Widget } from '@lumino/widgets';
 
 import { GridStack, GridStackNode, GridHTMLElement } from 'gridstack';
 
-import Cell from './components/cell';
-
 import 'gridstack/dist/gridstack.css';
 
+import Cell from './components/Cell';
```

Comment:
Looks like the CI failure is relevant: https://github.com/hbcarlos/voila-editor/runs/1181681805

Reply:
Which would most likely mean changing this line to:

```suggestion
import Cell from './components/cell';
```